src/Service/GithubApiService.php

```diff
@@ -368,7 +368,7 @@ public function enrichRepository(ObjectEntity $repository, array $repositoryArra
         }
 
         // If there is no component create one.
-        if ($repository->getValue('components') === false) {
+        if ($repository->getValue('components')->count() === 0) {
```

Comment:
Did you not change this from ->count to false in an earlier commit? (I might be wrong)🤔

Reply:
yes this is how it should be